docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -277,16 +277,45 @@ in the partition::
     DELETE FROM iceberg.testdb.customer_orders
     WHERE country = 'US' AND customer = 'Freds Foods'
 
-Rolling back to a previous snapshot
------------------------------------
+
+
+Snapshots
+---------
 
 Iceberg supports a "snapshot" model of data, where table snapshots are
 identified by an snapshot IDs.
 
 The connector provides a system snapshots table for each Iceberg table.  Snapshots are
-identified by BIGINT snapshot IDs.  You can find the latest snapshot ID for table
+identified by BIGINT snapshot IDs.  You can find the snapshot IDs for table
 ``customer_orders`` by running the following command::
 
+    SELECT snapshot_id FROM iceberg.testdb."customer_orders$snapshots" ORDER BY committed_at
+
+Snapshot queries
+^^^^^^^^^^^^^^^^
+
+The snapshots of an Iceberg tables can be easily queried by using the following command::
+
+   SELECT * FROM iceberg.testdb."customer_orders@8954597067493422955"
+
+In the command above, the special notation ``"customer_orders@8954597067493422955"`` is used to point to a
+specific snapshot of the ``customer_orders`` Iceberg table.
+
+
+The snapshot queries can be also used with special tables::
+
+
+     SELECT * FROM iceberg.testdb."customer_orders$data@8954597067493422955"
+     SELECT * FROM iceberg.testdb."customer_orders$partitions@8954597067493422955"
+     SELECT * FROM iceberg.testdb."customer_orders$manifests@8954597067493422955"
+     SELECT * FROM iceberg.testdb."customer_orders$files@8954597067493422955"
+
+
+Rolling back to a previous snapshot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can find the latest snapshot ID for table ``customer_orders`` by running the following command::
```

Comment:
You can determine the snapshot IDs for a table in the hidden metadata table ``$snapshots``. Use this table to determine the latest snapshot ID with a query similar to the following example::